lib/generateTypes.js

```diff
@@ -44,6 +43,11 @@ function replaceArrayTypes(type, name) {
         case "reviewers":
         case "comments":
             return "string[]";
+        case "body":
+          if (type === "Array") {
```

Comment:
There are a number of routes that use body for the property name which is sent as the request body, rather than body as a parameter name. I'll look to see if I can tidy that up later as I think some of those types are wrong too.